release-notes.txt

```diff
@@ -0,0 +1,237 @@
+BISDN Linux
+v4.5.0 Release Notes
+Date: 2022-03-15
+
+! New in this release
+- baseboxd:
+  - updated version to 1.11.1
+  - added support for Spanning Tree Protocol (STP, RSTP and MSTP) on bonded interfaces.
+  - added initial support for Broadcom KNET interfaces.
+- Linux kernel updated to version 5.15.22
+- iproute2: udpate to version 5.16
+
+! Experimental features
+- OF-DPA: initial support for creating and enabling Broadcom KNET interfaces.
+
+! Bug fixes
+- fixed bug related to terminaton MAC entries not being correctly removed on IP deletion. (https://github.com/bisdn/basebox/issues/362)
+- resolved STP issue where the initial STP state was missed, causing ports to forward/flood instead of blocking. (https://github.com/bisdn/bisdn.github.io/pull/136)
+- fixed an issue where any reachable ip neighbor was assumed to be routable even if a route did not exist. (https://github.com/bisdn/basebox/issues/364)
+- ofdpa: fixed 1G ports on PHY2/3 on platform as4630-54pe.
```

Comment:
what was fixed here?

Reply:
This was the polarity flip fix, the commit message reads:

> The polarity flip needs to be set for the 1g ports on PHY2/PHY3 to work, so set it for all 1G ports.

Reply:
```suggestion
- ES202 (AS4630-54PE): fixed an issue preventing some of the 1G ports from transmitting packets
```